src/routes/tutorial/[slug]/Loading.svelte

```diff
@@ -10,11 +10,16 @@
 
 	/** @type {string} */
 	export let status;
+
+	function webcontainer_is_supported() {
+		// @ts-ignore
+		return 'function' === typeof Atomics.waitAsync;
```

Comment:
Can we be sure that `Atomics` exists, or do we need an extra check like `typeof Atomics !== 'undefined'`?
Also: Yoda speak 😄

Reply:
You're right, this code would cause an error on browsers that don't support `Atomics`. But since https://github.com/sveltejs/learn.svelte.dev/issues/306#issuecomment-1538330568 gave me the production code from webcontainers.io, I would like to borrow it. I will therefore mark this as a solution.